src/libs/actions/Card.ts

```diff
@@ -0,0 +1,186 @@
+import Onyx from 'react-native-onyx';
+import * as API from '@libs/API';
+import * as Localize from '@libs/Localize';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {Response} from '@src/types/onyx';
+
+type Reason = 'damaged' | 'stolen';
+
+function reportVirtualExpensifyCardFraud(cardID: number) {
+    type ReportVirtualExpensifyCardFraudParams = {
+        cardID: number;
+    };
+
+    const parameters: ReportVirtualExpensifyCardFraudParams = {
+        cardID,
+    };
+
+    API.write('ReportVirtualExpensifyCardFraud', parameters, {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: ONYXKEYS.FORMS.REPORT_VIRTUAL_CARD_FRAUD,
+                value: {
+                    isLoading: true,
+                },
+            },
+        ],
+        successData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: ONYXKEYS.FORMS.REPORT_VIRTUAL_CARD_FRAUD,
+                value: {
+                    isLoading: false,
+                },
+            },
+        ],
+        failureData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: ONYXKEYS.FORMS.REPORT_VIRTUAL_CARD_FRAUD,
+                value: {
+                    isLoading: false,
+                },
+            },
+        ],
+    });
```

Comment:
Even in cases like this we should move `optimisticData`, `successData` & `failureData` out of API.write object and declare it above with proper type. This makes it easier for future developers to work on it :)
like this:
```suggestion

    const optimisticData: OnyxUpdate[] = [
            {
                onyxMethod: Onyx.METHOD.MERGE,
                key: ONYXKEYS.FORMS.REPORT_VIRTUAL_CARD_FRAUD,
                value: {
                    isLoading: true,
                },
            },
        ];
        
    const successData: OnyxUpdate[] = [
            {
                onyxMethod: Onyx.METHOD.MERGE,
                key: ONYXKEYS.FORMS.REPORT_VIRTUAL_CARD_FRAUD,
                value: {
                    isLoading: false,
                },
            },
        ];
        
    const failureData: OnyxUpdate[] = [
            {
                onyxMethod: Onyx.METHOD.MERGE,
                key: ONYXKEYS.FORMS.REPORT_VIRTUAL_CARD_FRAUD,
                value: {
                    isLoading: false,
                },
            },
        ];

    API.write('ReportVirtualExpensifyCardFraud', parameters, {
        optimisticData,
        successData,
        failureData,
    });
```